iogt/settings/base.py

```diff
@@ -512,6 +512,6 @@
     'VAPID_ADMIN_EMAIL': os.getenv('VAPID_ADMIN_EMAIL'),
 }
 
-ENABLE_FE_COMMENTS_MODERATION = True if os.getenv('ENABLE_FE_COMMENTS_MODERATION') in ['True', 'true', '1'] else False
-
+COMMENTS_COMMUNITY_MODERATION = os.getenv('COMMENTS_COMMUNITY_MODERATION') == 'enable'
+COMMENTS_COMMUNITY_MODERATOR_CLASS = os.getenv('COMMENTS_COMMUNITY_MODERATOR_CLASS')
```

Comment:
It does not make sense to call this "community moderation" because the intention is to delegate to an external, automatic service for moderation - the community has nothing to do with it. To avoid confusion, let's call this `COMMENTS_MODERATOR_SERVICE`.